python/ribasim/tests/test_validation.py

```diff
@@ -14,18 +14,16 @@
 from shapely import MultiPolygon, Point, Polygon
 
 
-def test_duplicate_link(basic):
+def test_multiple_outflows(basic):
     model = basic
     with pytest.raises(
         ValueError,
-        match=re.escape(
-            "Links have to be unique, but link with from_node_id 16 to_node_id 1 already exists."
-        ),
+        match=re.escape("Node 16 can have at most 1 flow link outneighbor(s) (got 1)"),
     ):
         model.link.add(
             model.flow_boundary[16],
             model.basin[1],
-            name="duplicate",
+            name="multiple-outflows",
         )
```

Comment:
I tried to come up with a `link.add` that triggers the unique edges validation first, but I don't think we can trigger it anymore because we'd need a from_node that supports multiple outputs and to_node that supports multiple inputs, that are allowed to be connected. I don't think that exists anymore.